Bump core dependency (#35463)
Azure/azure-sdk-for-net: sdk/core/Azure.Core.Experimental/CHANGELOG.md

# Release History

## 0.1.0-preview.26 (2023-04-10)

### Features Added

- Added basic debugger support for DynamicData

> 

Reviewer: ```suggestion
### Breaking Changes

- Removed `DynamicJson` type, and moved its functionality into `DynamicData`.
- Renamed `DynamicJsonNameMapping` enum to `DynamicDataNameMapping`.
- Renamed `DynamicJsonProperty` to `DynamicDataProperty`.
- Renamed `ToDynamic()` extension method on `BinaryData` to `ToDynamicFromJson()`.
- Removed `DynamicJson.ArrayEnumerator` and `DynamicJson.ObjectEnumerator` types.
```

### Bugs Fixed

> - Use `DynamicJsonOptions` in child elements, arrays, and object enumerators.

Reviewer: ```suggestion
- Use root `DynamicJsonOptions` in child elements, arrays, and object enumerators.
```

Author: I think you added this in your other comment :)

# Release History

## 0.1.0-preview.26 (2023-04-10)

### Features Added

- Added basic debugger support for DynamicData
### Breaking Changes

> - Removed `DynamicJson` type, and moved its functionality into `DynamicData`.

Reviewer: ```suggestion
- Removed `DynamicJson` type, and moved its functionality into `DynamicData`.
- Sealed the `DynamicData` type.
```

# Release History

> ## 0.1.0-preview.26 (2023-04-10)

Reviewer: ```suggestion
## 0.1.0-preview.26 (2023-04-10)

### Features Added

- Added basic debugger support for DynamicData
```

### Bugs Fixed

> - Use specified `DynamicJsonNameMapping` in array and object enumerators.

Reviewer: ```suggestion
- Use `DynamicJsonOptions` in child elements, arrays, and object enumerators.
```
